downloads/platform.html

```diff
@@ -24,11 +24,34 @@ <h2 id='Windows'>Windows</h2>
         Julia is available for Windows 7 and later, both 32 bit and 64 bit.
         <ol>
           <li>Download the Windows julia.exe installer for your platform. 32-bit julia works on both x86 and x86_64. 64-bit julia will only run on 64-bit Windows (x86_64).
-          <li>Run the downloaded program to extract julia
-          <li>Double-click the julia shortcut in the unpacked folder to start julia
+          <li>Run the downloaded program to extract Julia.
+          <li>At the "Choose Installation directory" step, copy the Path in the "Destination Folder" into a Notepad or similar file for future reference, and finish installation.
+          <li>At this point, you can decide whether you want to invoke Julia by simply typing <code>julia</code> in the command line, or paste the entire path you saved in the Notepad. To invoke Julia by simply typing <code>julia</code> in the command line, perform the following steps.
         </ol>
+        <p>
+          If on Windows 10,
+          <ol>
+            <li>Hit the Windows Key, and type in "env", and select "Edit the system environment variables"
+            <li>Click the "Environment Variables..." button.
+            <li>Under the "System Variables" section (the lower half), find the row with "Path" in the first column, and click edit.
+            <li>The "Edit environment variable" UI will appear. Here, click "New", and paste in the text you copied into the Notepad.
+            <li>Hit OK and dismiss all the dialogs. You can now run Julia from the command line!
+          </ol>
+        </p>
+        <p>
+          Else, if on Windows 7 or 8,
+          <ol>
+            <li>Go to the Windows Desktop and right click on "My Computer" and hit "Properties"
```

Comment:
`My Computer` icon might not be available for some users since they should be enabled first. So, a more general solution here too would be useful.

Reply:
True, but I don’t have a Windows 7 OS around me. Maybe a Windows 7 user can change this? Anyways, this was the only method I found for Windows 7, so I added it.